package.json

```
@@ -18,6 +18,14 @@
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
     "@types/uuid": "^10.0.0",
+    "@uppy/aws-s3": "^4.1.0",
+    "@uppy/core": "^4.2.0",
+    "@uppy/dashboard": "^4.1.0",
+    "@uppy/drag-drop": "^4.0.2",
+    "@uppy/file-input": "^4.0.1",
+    "@uppy/golden-retriever": "^4.0.0",
```

It doesn't look like this is used?

---

I'll remove. The other packages are needed  
```suggestion
    "@uppy/golden-retriever": "^4.0.0",
```